src/index.html

```diff
@@ -5,11 +5,24 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
 
     <link rel="stylesheet" href="css/main.css" />
-    <script src="js/main.js" />
+
+    <link href="https://fonts.googleapis.com/css2?family=Rubik:wght@400;700&display=swap" rel="stylesheet"> 
```

Comment:
Could remove this font import please? I know it will look dumb for a bit, but there is a different font that we want to import.